src/current/_includes/molt/migration-prepare-database.md

```diff
@@ -1,15 +1,141 @@
+#### Create migration user on source database
+
+Create a dedicated migration user (e.g., `MIGRATION_USER`) on the source database. This user is responsible for reading data from source tables during the migration.
+
 <section class="filter-content" markdown="1" data-scope="postgres">
-Ensure that the PostgreSQL database is configured for replication. Enable logical replication by setting `wal_level` to `logical` in `postgresql.conf` or in the SQL shell. For example:
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE USER MIGRATION_USER WITH PASSWORD 'password';
+~~~
+
+Grant the user privileges to connect, view schema objects, and select the tables you migrate.
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-ALTER SYSTEM SET wal_level = 'logical';
+GRANT CONNECT, RESOURCE ON DATABASE source_database TO MIGRATION_USER;
```

Comment:
Is this meant to be for a specific dialect or all databases? Agreed with Florence's point here. This is something we want to specify as for Oracle.